content/courses/connecting-to-offchain-data/oracles.md

```diff
@@ -744,7 +743,7 @@ mod burry_oracle_program {
 
     use super::*;
 
-    pub fn deposit(ctx: Context<Deposit>, escrow_amt: u64, unlock_price: u64) -> Result<()> {
+    pub fn deposit(ctx: Context<Deposit>, escrow_amt: u64, unlock_price: f64) -> Result<()> {
         deposit_handler(ctx, escrow_amt, unlock_price)
```

Comment:
The old code has a couple of unnecessary wrapper functions. Also, technically, the function that receives the incoming instruction first is the instruction handler.

Reply:
Also `amt` is a bad variable name, see https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#code